gapic/src/main/com/google/gapid/util/URLs.java

```diff
@@ -18,4 +18,6 @@
 public interface URLs {
   public static final String FILE_BUG_URL =
       "https://github.com/google/gapid/issues/new?template=standard-bug-report-for-gapid.md";
+  public static final String DEVICE_COMPATIBILITY =
```

Comment:
Could you please file a bug about adding device compatibility doc and assign it to me? And then add a TODO with the bug id here.